packages/in_app_purchase/tizen/src/billing_manager.h

```diff
@@ -12,10 +12,14 @@
 
 #include <iostream>
 #include <map>
+#include <mutex>
```

Comment:
nit: As you know, we try to follow google c++style. https://google.github.io/styleguide/cppguide.html#Include_What_You_Use
These header declare should also be applied to bilinging_manager.cc.